doc/visualizations.rst

```diff
@@ -8,37 +8,81 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and
+input data (`X`, `y`).
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`). Using `from_predictions`
+avoids to recompute the predictions, but does not automatically resolve some
+ambiguities. The reason being that the user needs to know which column corresponds
+to the positive label (in this case, `y_pred`).
```

Comment:
Note that this is not the case for decision function where a 1d array is returned (you need to multiply by -1, if you wish to switch `pos_label`). This is why I used the wording I suggested. If you wish, you can go into detail about the adaption required for both proba and decision function, or simply leave as the wording i suggested.

Reply:
The second sentence does not seem to naturally follow on from the first. The first talks about `from_predictions` not automatically solving ambiguities, and then the second explains what the user needs to do.

i.e. if you said in your first sentence, using `from_prediction` means that the user needs to resolve ambiguities, then the second sentence makes more sense.